src/main/java/org/opentripplanner/middleware/models/TripRequest.java

```diff
@@ -36,24 +38,27 @@
      */
     public String toPlace;
 
-    /**
-     * Query params. Query parameters influencing trip.
-     */
-    //TODO: This could be the request parameters returned as part of the plan response. Would be POJO based instead of just text.
-    public String queryParams;
+    /** A dictionary of the parameters provided in the request that triggered this response. */
+    public HashMap<String, String> requestParameters;
```

Comment:
Note for self: Ooooh this change needs to be reflected in https://github.com/opentripplanner/otp-react-redux/pull/329.